torch/ao/quantization/backend_config/observation_type.py

```diff
@@ -11,3 +11,6 @@ class ObservationType(Enum):
     # on qconfig.activation
     # example: torch.cat, maxpool
     OUTPUT_SHARE_OBSERVER_WITH_INPUT = 1
+    # this means the output is never observed
+    # example: x.shape, x.size
+    OUTPUT_NOT_OBSERVED = 2
```

Comment:
if both input and output are not observed maybe we want to rename this to `INPUT_OUTPUT_NOT_OBSERVED`?

Reply:
I see, sure, that sounds good

Reply:
thanks, can we also document if we will dequantize the Tensor before call tensor.size/tensor.shape here as well? see https://github.com/pytorch/pytorch/pull/83784#issuecomment-1224928070